bundles/statistics/statsgrid2016/resources/locale/en.js

```diff
@@ -173,7 +173,8 @@ Oskari.registerLocalization({
             'myIndicatorRegionselect': 'Regionselect cannot be empty.',
             'myIndicatorDatasource': 'Datasource is empty.',
             'cannotDisplayAsSeries': 'Indicator cannot be inspected as a series.',
-            'noDataForIndicators': 'Service did not return data for {indicators, plural, one {the indicator} other {indicators}}'
+            'noDataForIndicators': 'Service did not return data for {indicators, plural, one {the indicator} other {indicators}}',
+            'noAllDataForIndicators': 'Service did not return all data for {indicators, plural, one {the indicator} other {indicators}}'
```

Comment:
The key for this could be renamed to onlyPartialDataForIndicators or incompleteDataForIndicators or something more descriptive as we got some but not all the data that was requested? Like the user requested 1990-2019 but we only got 2000-2018

Reply:
Updated key to onlyPartialDataForIndicators as suggested.